gdcdictionary/schemas/treatment.yaml

```diff
@@ -520,6 +664,7 @@ properties:
       - Wrist
       - Unknown
       - Not Reported
+      - Locoregional Site
```

Comment:
prob. need to move this one upwards for alphabeticalization

Reply:
darn how did I miss this one and some others 😢 ok fixed, thanks for the check.